### DISCUSSION TITLE
Tapping a pull request in notifications sometimes leads to the Files view

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#70546] [Mobile] [answered]
GitHub Mobile beta for Android

Tapping a pull request in notifications

![Screenshot_20231019-075410.png](https://github.com/community/community/assets/2119212/81a1317a-5f99-445e-a673-ffbda96c2ac9)

Sometimes leads to the Files view.
![Screenshot_20231019-075751.png](https://github.com/community/community/assets/2119212/eee76f66-b152-4026-8ecb-f7b78370afdd)

If you tapped on "1 commit". But, it looks like a row, not a thing with distinctly tappable elements, and I believe the entire row highlights when this tap happens. 

It should lead to the PR view, just as it would in a web browser.

**Comment by elisealix22** [comment, marked as answer]
Hi @jsoref , thank you so much for reporting this! I believe we are currently opening the Files Changed view on top of the PR view for those notifications, but I tend to agree with you that it is not what I would expect looking at that notification. I opened an internal issue to track this and we will discuss it further as a team! Let us know if you have any other suggestions in the meantime.

**Reply by jsoref** [reply]
You are correct, it's an on-top, and while that means I can get where I want to go, I'm fairly confident it violates HIG or similar.

The fact that (on Android -- but not iOS!!) I can't go to Notifications w/o walking all the way back up the Back stack makes me hate things more the more times they add more than one item to the stack.
